content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -21,7 +21,7 @@ operation.
   in the <abbr>workspace</abbr> will only contain links to the data files in the
   cache (refer to
   [Large Dataset Optimization](/doc/user-guide/large-dataset-optimization). See
-  `dvc config cache` for related configuration options, including changing the
+  `dvc config cache` for related configuration options, including changing
```

Comment:
I think this was intentional. Meant to be read "changing the DVC config cache for related config options...".

I'm not sure "changing `dvc config cache` for related..." is better. Could be "using `dvc config cache` to change the related..."